paper/paper.md

```diff
@@ -102,6 +107,7 @@ Augur can be installed from PyPI ([nextstrain-augur](https://pypi.org/project/ne
 # Acknowledgments
 
 Thank you to all of [the open source community members who have contributed to Augur](https://github.com/nextstrain/augur/graphs/contributors).
+Specifically, we thank Eric Danielson, Eddie Lebow, Barney Potter, Ryan Grout, Sai Kiran Kollapudi, Mingye Wang, Carol Willing, Louise Moncla, Thomas Caswell, Sidney Bell, Terry Jones, Christian Clauss, Julien Bordellier, Gytis Dudas, Cameron Devine, Samuel Zhang, Akshay Subramanian, Christopher Tomkins-Tinch, Danielle Kain, Pierre Barrat-Charlaix, Rhys Kidd, Chris Woszczak, Tony Tung, Mathias Walter, and Zachary Sailer.
```

Comment:
FWIW, I was surprised to see this list in non-alphabetical order.

Reply:
That's fair. This order is basically as random as alpha. ;)